pvlib/pvsystem.py

```diff
@@ -1908,33 +1908,36 @@ def sapm_celltemp(poa_global, wind_speed, temp_air,
     return pd.DataFrame({'temp_cell': temp_cell, 'temp_module': temp_module})
 
 
-def pvsyst_celltemp(poa_global, wind_speed, temp_air, eta_m=0.1,
-                    alpha_absorption=0.9, loss_factors="freestanding"):
+def pvsyst_celltemp(poa_global, temp_air, wind_speed=1.0, eta_m=0.1,
+                    alpha_absorption=0.9, model_params='freestanding'):
     """
     Calculate cell temperature using an emperical heat loss factor model
     as implemented in PVsyst.
 
-    The heat loss factors represent the combined effect of convection,
-    radiation and conduction, and their values are experimentally determined.
+    The heat loss factors provided through the 'model_params' argument
+    represent the combined effect of convection, radiation and conduction, 
+    and their values are experimentally determined.
 
     Parameters
     ----------
     poa_global : numeric
         Total incident irradiance in W/m^2.
 
-    wind_speed : numeric
-        Wind speed in m/s at a height of 10 meters.
-
     temp_air : numeric
         Ambient dry bulb temperature in degrees C.
 
+    wind_speed : numeric
```

Comment:
format is `wind_speed : numeric, default 1.0`  Remove editorial parenthetical.  I'm OK with the default wind_speed value of 1.0.